ballerina/Ballerina.toml

```diff
@@ -1,8 +1,8 @@
 [package]
-distribution = "2201.3.0"
+distribution = "2201.8.0"
 org = "ballerinax"
 name = "aws.simpledb"
-version = "2.0.0"
+version = "3.0.0"
```

Comment:
Any reason to bump the major version, instead of the minor version?

Reply:
Converted to the major version by mistake. Updated in [a1ef477](https://github.com/ballerina-platform/module-ballerinax-aws.simpledb/pull/28/commits/a1ef477c8cba12962c5567ca14e87581d213326f)